core/org.wso2.carbon.ui/src/main/resources/web/dialog/js/dialog.js

```diff
@@ -227,7 +227,7 @@ CARBON.showConfirmationDialog = function(message, handleYes, handleNo, closeCall
  * @param {String} message to display
  * @return {Boolean}
  */
-CARBON.showPopupDialog = function(message, title, windowHight, okButton, callback, windowWidth) {
+CARBON.showPopupDialog = function(message, title, windowHeight, okButton, callback, windowWidth) {
```

Comment:
Need to rename the usages of this variable (L267,269)..

Reply:
fixed with c7f3e5b